draft-ietf-quic-transport.md

```diff
@@ -163,7 +163,8 @@ Connection:
 
 Connection ID:
 
-: The 64-bit unsigned number used as an identifier for a QUIC connection.
+: An opaque identifier that is used to identify a QUIC connection.  Each
```

Comment:
This sentence sounds incomplete now. How about "to identify a QUIC connection at an endpoint"?